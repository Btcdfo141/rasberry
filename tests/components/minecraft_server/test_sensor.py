"""Tests for the Minecraft Server sensor platform."""
