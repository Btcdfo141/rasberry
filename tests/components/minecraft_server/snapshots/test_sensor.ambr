# serializer version: 1
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response4-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '5',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response5-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '3',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response6-sensor.minecraft_server_motd]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World Message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_motd',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response6-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response7-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Version',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response8-sensor.minecraft_server_map_name]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Map name',
      'icon': 'mdi:map',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_map_name',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Map Name',
  })
# ---
# name: test_sensor[bedrock_mock_config_entry-BedrockServer-status_response9-sensor.minecraft_server_game_mode]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Game mode',
      'icon': 'mdi:cog',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_game_mode',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Game Mode',
  })
# ---
# name: test_sensor[java_mock_config_entry-JavaServer-status_response0-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '5',
  })
# ---
# name: test_sensor[java_mock_config_entry-JavaServer-status_response1-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'players_list': list([
        'Player 1',
        'Player 2',
        'Player 3',
      ]),
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '3',
  })
# ---
# name: test_sensor[java_mock_config_entry-JavaServer-status_response2-sensor.minecraft_server_motd]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World Message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_motd',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor[java_mock_config_entry-JavaServer-status_response2-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor[java_mock_config_entry-JavaServer-status_response3-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Version',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response4-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '5',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response5-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '3',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response6-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response7-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Version',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response8-sensor.minecraft_server_map_name]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Map name',
      'icon': 'mdi:map',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_map_name',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Map Name',
  })
# ---
# name: test_sensor_update[bedrock_mock_config_entry-BedrockServer-status_response9-sensor.minecraft_server_game_mode]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Game mode',
      'icon': 'mdi:cog',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_game_mode',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Game Mode',
  })
# ---
# name: test_sensor_update[java_mock_config_entry-JavaServer-status_response0-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '5',
  })
# ---
# name: test_sensor_update[java_mock_config_entry-JavaServer-status_response1-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'players_list': list([
        'Player 1',
        'Player 2',
        'Player 3',
      ]),
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': '3',
  })
# ---
# name: test_sensor_update[java_mock_config_entry-JavaServer-status_response2-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy MOTD',
  })
# ---
# name: test_sensor_update[java_mock_config_entry-JavaServer-status_response3-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'Dummy Version',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response4-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response5-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response6-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response7-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response8-sensor.minecraft_server_map_name]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Map name',
      'icon': 'mdi:map',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_map_name',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[bedrock_mock_config_entry-BedrockServer-status_response9-sensor.minecraft_server_game_mode]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Game mode',
      'icon': 'mdi:cog',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_game_mode',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[java_mock_config_entry-JavaServer-status_response0-sensor.minecraft_server_latency]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Latency',
      'icon': 'mdi:signal',
      'unit_of_measurement': <UnitOfTime.MILLISECONDS: 'ms'>,
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_latency',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[java_mock_config_entry-JavaServer-status_response1-sensor.minecraft_server_players_online]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Players online',
      'icon': 'mdi:account-multiple',
      'unit_of_measurement': 'players',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_players_online',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[java_mock_config_entry-JavaServer-status_response2-sensor.minecraft_server_world_message]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server World message',
      'icon': 'mdi:minecraft',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_world_message',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
# name: test_sensor_update_failure[java_mock_config_entry-JavaServer-status_response3-sensor.minecraft_server_version]
  StateSnapshot({
    'attributes': ReadOnlyDict({
      'friendly_name': 'Minecraft Server Version',
      'icon': 'mdi:numeric',
    }),
    'context': <ANY>,
    'entity_id': 'sensor.minecraft_server_version',
    'last_changed': <ANY>,
    'last_updated': <ANY>,
    'state': 'unavailable',
  })
# ---
